src/infection-model/seir.tsx

```diff
@@ -57,6 +57,26 @@ enum ageGroupIndex {
   __length,
 }
 
+// model constants
+// days from virus exposure to infectious/contagious state
+const dIncubation = 2;
+// days from virus exposure to symptoms
+const dSymptoms = 5.1;
+// days in infectious period
+const dInfectious = 4.1;
+// days from end of infectious period to end of virus
+const dRecoveryMild = 9.9;
+// days from end of infectious period to hospital admission
+const dHospitalLag = 2.9;
+// days from hospital admission to hospital release (non-fatality scenario)
+const dHospitalRecovery = 22;
+// days from hospital admission to deceased (fatality scenario)
+const dHospitalFatality = 8.3;
+// probability case will be severe enough for the hospital
+const pSevereCase = 0.26;
+// factor for inferring exposure based on confirmed cases
+const ratioExposedToInfected = dIncubation / dSymptoms;
```

Comment:
This value (2 / 5.1) is different from what was hard-coded below... Was this a bug or another misunderstanding on my part?

Reply:
ugh good catch! sloppy work on my part, I put in the wrong denominator. stand by...